bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/handler/BlueZBridgeHandler.java

```diff
@@ -88,8 +88,13 @@ public BlueZBridgeHandler(Bridge bridge) {
 
     @Override
     public void initialize() {
+        BluetoothManager manager;
         try {
-            BluetoothManager.getBluetoothManager();
+            manager = BluetoothManager.getBluetoothManager();
+            if (manager == null) {
```

Comment:
Looking at the [source](https://github.com/intel-iot-devkit/tinyb/blob/master/java/BluetoothManager.java), it doesn't seem `getBluetoothManager` can return 'null'.

Reply:
I did receive a `null`. Maybe it's because the native code was not loaded properly.

Reply:
Ok, it's a native library, so perhaps it is better to check for null regardless. Still weird, wonder what could've gone wrong.